src/components/Annotation.tsx

```diff
@@ -90,7 +172,11 @@ export default function Annotation({
     draggable,
     enabled,
     visible,
+
     clusteringIdentifier,
+    displayPriority,
```

Comment:
You should make sure you handle the new string values for `displayPriority` correctly (a correction function in `parameters.ts` could be helpful.

Reply:
What do you mean by an "correction function"? Like ensuring it's between 0 and 1000 but then we are not talking about string values... also ts should check that it's a number (not necessarily within range) or one of the allowed strings.

Reply:
So I had a look into `parameters.ts` and think you are asking to introduce enum values instead of the current string behavior maybe? So we would have enums for `low`, `high` and `required`? Could be a good idea, I agree!

Reply:
Sorry, I meant conversion, not correction. We don’t need to do parameter validation here, but when the user sets `"high"` as the parameter value we should send the value `mapkit.Annotation.DisplayPriority.High` to MapKit JS (and do the same for the two other string values). I don’t think an enum is necessary here.